app/src/renderer/components/governance/ModalPropose.vue

```diff
@@ -59,17 +59,11 @@
         type="text"
         readonly="readonly"
       />
-      <tm-field
-        id="amount"
-        :max="balance"
-        :min="0"
-        v-model="amount"
-        type="number"
-      />
+      <tm-field id="amount" :min="0" v-model="amount" type="number" />
       <tm-form-msg
         v-if="!$v.amount.between && amount > 0"
         :max="$v.amount.$params.between.max"
-        :min="$v.amount.$params.between.min"
+        :min="1"
```

Comment:
but in this case on line 62 we are saying that the minimum is 0 and in the error message the minimum is 1

Reply:
or is the `1` referring to the first parameter in line 186? 

In both cases I think is safer to use the actual parameters of the validation function in the error message, to avoid having to remember to update both places if we change this in the future.

Reply:
you mean updating the validation minLength to be 1 ?

Reply:
> This should be just the value to put in the string we will see in the error message

So 1 ?

Reply:
I am on @sabau side here. What was your intent while changing this?